cellfinder_core/classify/classify.py

```diff
@@ -23,13 +24,24 @@ def main(
     model_weights,
     network_depth,
     max_workers=3,
+    *,
+    callback=None,
 ):
-
+    """
+    Parameters
+    ----------
+    callback : Callable[int], optional
```

Comment:
Why not add the typing above, i.e. `Optional[Callable[int]]`? I know the others aren't typed but doesn't mean we can't make a start

Reply:
Isn't `pre-commit` meant to tell me to do this 😉  Added.

Reply:
No `mypy` by default will pass if there is no typing, it only checks that the typing there is correct. There may be a way to force this, but it will be a lot of work to add all the typing in one go. So I think it's good to try and make sure that all new code has typing